testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/SigningInTest.java

```diff
@@ -62,6 +64,9 @@
 /**
  * @author Vaclav Muzikar <vmuzikar@redhat.com>
  */
+@EnableFeature(value = Profile.Feature.ACCOUNT2, skipRestart = true)
+@EnableFeature(value = Profile.Feature.ACCOUNT_API, skipRestart = true)
+@EnableFeature(value = Profile.Feature.WEB_AUTHN, skipRestart = true)
```

Comment:
I believe we don't need to enable WebAuthn for the whole test class. This affects only:
https://github.com/keycloak/keycloak/blob/26a78a1eb61b4e2b293bcf98d06fe78713492b70/testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/SigningInTest.java#L246-L254

Reply:
Scratch that! I completely forgot we configure WebAuthn before running the whole test class. This approach is therefore correct.

Reply:
There are more tests, which depend on the WebAuthn feature. So, I think, it would be better to keep for whole class.
f.e. `categoriesTest`,`setUpLinksTest`,...

Reply:
@mabartos You're, of course, correct. I almost forgot what's in the test I wrote. I guess it's monday... :D